Diff-Word.ps1

```diff
@@ -3,6 +3,9 @@ param(
     $ChangedFileName
 )
 
+$BaseFileName = Resolve-Path $BaseFileName | Select-Object -first 1 | Select -ExpandProperty "Path"
+$ChangedFileName = Resolve-Path $ChangedFileName | Select-Object -first 1 | Select -ExpandProperty "Path"
```

Comment:
Could you please make it more concise? I think `(Resolve-Path $BaseFileName).Path` should be enough.